### PR TITLE
Feature/testing

### PR DESCRIPTION
**Pull request by Nasa28**
In This project I implemented testing using Jest and Testing-Library  for ;
1. calculate.js
2. NavBar component
3. Home component
4. Quotes component
5. Calculator component

### CODE REVIEW

File: src/component/__test__/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  it('Returns the Result of Division operation', () => {
+    const data = { total: '10', next: '10', operation: '÷' };
+    expect(calculate(data, '÷')).toEqual({
+      total: '1',
+      next: '10',
+      operation: '÷',
+    });
+  });
+});
+
```
Comment (rindrajosia) [inline src/component/__test__/calculate.test.js]: - [x] [REQUIRED] I got an error when I divide numbers by zero. It will be nice if you fix it and add a test unit for that case.

File: src/component/Pages/Calculator/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    setData(calculate(data, buttonName));
+  };
+  const { total, next } = data;
+  const currentResult = total || next;
+  return (
+    <>
+      <div className="main">
+        <h3>Lets Do Some Maths!</h3>
+        <div className="app">
+          <Display result={currentResult} />
```
Comment (rindrajosia) [inline src/component/Pages/Calculator/Calculator.js:23]: - [x] [REQUIRED] For user experience, I suggest that you change the `value displayed in the Display component` when the user clicks on one operator. For example, in this image, after clicking on `X` and once the user enters a new value (here it is: 9), the value displayed in the `Display component` should be 9 not 7

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:


## Required Changes ♻️

- [x] Please make sure that information in your readme is correct. For example, your MIT link is broken, please fix it.
- [x] It will be nice if you add tests units for negative cases. The aim is to make sure that your APP works correctly.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- to update your live demo.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by rukundoeric — APPROVED** [review]
Hi @Nasa28,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
